### PR TITLE
using scopegen output version 2

### PR DESCRIPTION
**Pull request by favadi**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -11,3 +11,5 @@ require (
 	google.golang.org/genproto v0.0.0-20190708153700-3bdd9d9f5532
 	google.golang.org/grpc v1.21.1
 )
+
+replace github.com/gunk/gunk => github.com/favadi/gunk v0.1.1-0.20200121080516-e0d382f4de09
```
Comment (nkristianto) [inline go.mod]: don't forget to replace this.

**Review by nkristianto — APPROVED** [review]
LGTM. thank for this.

File: go.mod (hunk 1 above)
Comment (favadi) [reply go.mod]: removed, as the gunk PR merged.

**Review by jwangsadinata — APPROVED** [review]
LGTM, thanks so much for this @favadi

**Pull request merged**